apps/src/code-studio/progressRedux.ts

```diff
@@ -353,6 +353,16 @@ export function sendSuccessReport(appType: string): ProgressThunkAction {
   };
 }
 
+// The user has attempted the level (as defined by the app type). Currently only used by Lab2 labs.
+export function sendInProgressReport(
```

Comment:
nit `sendProgressReport` or even just `reportProgress` since this can accept any type of `result` now?

Also update comment?

Reply:
I'll go with `sendProgressReport` so it matches in style `sendSuccessReport`. And updated comment - thanks!